### ISSUE TITLE
Startup sound, and Mode beeps do not send over MIDI USB

### ISSUE THREAD

**Issue by Ekolide** [cardonabits/haxo-rs#37]
Hi.

When starting the Haxophone there is a startup sound played to indicate when the instrument is ready to be played. This sound plays through Fluidsynth, but not MIDI USB. The instrument changing mode is similar; it does not send note information to preview the current instrument, or beep when entering or exiting the mode. The transposition mode does not send any beeps either.

I think this would be a useful thing to have access to for cohesion and navigation, but there should also be an way to turn these sounds off for performance. For example by hitting all three RH palm keys while in the instrument or transposition mode. When hitting all RH palm keys and there is no sound, the setting is off, when hitting all RH palm keys and it plays a beep, the setting is on.

Of course, there's the element of not knowing what sounds will actually trigger when sending this note information, since the user will be using their own sounds and MIDI routing. I'm interested to hear opinions on this.

Thanks!

**Comment by jcard0na** [comment]
When I implemented USB MIDI I decided not to send any "control sounds" via USB because I thought it would disrupt the (my?) main use case:  transcribing to a score what is played on the haxophone.  I would not oppose to someone implementing this feature, as long as it is off by default, and can be turn on either at compilation time (e.g. `--feature control-sounds-to-midi'` or during playtime via some key press combination, like the one you suggest).

**Comment by Ekolide** [comment]
I agree, I'm not convinced that these sounds are always needed, your suggestions makes sense.
